src/components/SharedComponents/SwitchComponent.stories.mdx

```diff
@@ -0,0 +1,10 @@
+import { Meta, Story } from '@storybook/addon-docs/blocks';
+import SwitchComponent from './SwitchComponent';
+
+<Meta title="Shared Components/Switch Component" argTypes={{ onChange: { action: 'changed' } }} />
```

Comment:
FWIW- I saw that you have some global argTypesRegex thing: https://github.com/covid-projections/covid-projections/blob/f9c78b6eb942f585dddccdcb6aed3f486125b79a/.storybook/preview.js#L11

Not sure if I can/should be integrating with that somehow.

Reply:
This is added by default by Storybook, I'm not sure if it does anything 🤷‍♂️